playground/src/experiments/experiments.scss

```diff
@@ -0,0 +1,32 @@
+@import 'mixins.scss';
+@import 'components/atoms/_text.scss';
+
+#experiments-view {
+
+  .fly-in-content {
+    overflow: hidden;
+
+    .experiments {
+      &-list {
```

Comment:
What I don't like is that this is creating a very overqualified selector:  `#experiments-view .fly-in-content .experiments-list`